README.md

```diff
@@ -44,6 +44,63 @@ Include the download centre nav-menu:
     ...
 ```
 
+## WS mode
+Package can be used with websockets to provide asynchronous communication
+between frontend and backend.
+If package will be used in websockets mode additional settings must be applied.
+* Inside common settings `WS_MODE` must be toggled on, and master url need to be provided.
```

Comment:
```suggestion
* Inside common settings `WS_MODE` must be toggled on.
```
Then could also mention here that the channel name can be set as well, or left as the default.

Reply:
c3c1419